x-pack/plugins/security_solution/public/common/components/event_details/cti_details/threat_details_view.tsx

```diff
@@ -125,31 +103,115 @@ const buildThreatDetailsItems = (enrichment: CtiEnrichment) =>
+    {enrichments
+      .sort((a, b) => getFirstSeen(b) - getFirstSeen(a))
+      .map((enrichment, index) => (
+        <EnrichmentAccordion
+          enrichment={enrichment}
+          index={index}
+          enrichmentsLength={enrichments.length}
+        />
+      ))}
+  </>
+);
+
 const ThreatDetailsViewComponent: React.FC<{
   enrichments: CtiEnrichment[];
 }> = ({ enrichments }) => {
-  const sortedEnrichments = enrichments.sort((a, b) => getFirstSeen(b) - getFirstSeen(a));
+  const {
+    [ENRICHMENT_TYPES.IndicatorMatchRule]: indicatorMatches,
+    [ENRICHMENT_TYPES.InvestigationTime]: threatIntelEnrichments,
+    undefined: matchesWithNoType,
+  } = groupBy(enrichments, 'matched.type');
 
   return (
     <>
       <EuiSpacer size="m" />
-      {sortedEnrichments.map((enrichment, index) => {
-        const { id, field, provider, type, value } = getEnrichmentIdentifiers(enrichment);
-
-        return (
-          <Fragment key={id}>
-            <ThreatDetailsHeader field={field} provider={provider} value={value} type={type} />
-            <EuiSpacer size="m" />
-            <StyledEuiInMemoryTable
-              columns={columns}
-              compressed
-              data-test-subj={`threat-details-view-${index}`}
-              items={buildThreatDetailsItems(enrichment)}
+      {indicatorMatches && (
+        <div data-test-subj={'threat-match-detected'}>
+          <EuiTitle size="xxxs">
+            <h5>{i18n.INDICATOR_TOOLTIP_TITLE}</h5>
+          </EuiTitle>
+          <EuiSpacer size="xs" />
+          <EuiText size="xs">
+            <FormattedMessage
+              id="xpack.securitySolution.alertDetails.threatDetails.threatMatchSubtitle"
+              defaultMessage="We have found {totalCount, plural, one {# field value} other {# field values}} matched a threat intelligence indicator with a rule you created."
+              values={{
+                totalCount: indicatorMatches.length,
+              }}
             />
-            {index < sortedEnrichments.length - 1 && <EuiSpacer size="m" />}
-          </Fragment>
-        );
-      })}
+          </EuiText>
+          <EuiSpacer size="s" />
+          <EnrichmentSection enrichments={indicatorMatches} />
+        </div>
+      )}
+      {threatIntelEnrichments && (
+        <div data-test-subj={'enriched-with-threat-intel'}>
+          {indicatorMatches && <EuiSpacer size="l" />}
+          <>
+            <EuiTitle size="xxxs">
+              <h5>{i18n.INVESTIGATION_TOOLTIP_TITLE}</h5>
+            </EuiTitle>
+            <EuiSpacer size="xs" />
+            {/* TODO: Date form */}
+            <EuiText size="xs">
+              <FormattedMessage
+                id="xpack.securitySolution.alertDetails.threatDetails.investigationSubtitle"
+                defaultMessage="We have found {totalCount, plural, one {# field value} other {# field values}} has additional information available from threat intelligence sources we searched in the past 30 days by default."
```

Comment:
This sentence is reading odd when there are multiple values: Either the '...has additional' needs to also reflect the count logic, or I wonder if we can simply not worry about mentioning the count here at all since it's more prominently displayed in the tab itself.  So maybe something simple like: `We have found additional information available from threat intelligence sources...` and we can then remove all the conditional logic here?

Reply:
will bring up this sentence with product during our CTI meeting today, thanks for pointing it out!